.github/workflows/ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
         with:
           shared-key: "sanity"
-          save-if: ${{ github.ref == 'refs/pull/3358/merge' }}
+          save-if: ${{ github.ref_name == 'testnet_19' }}
```

Comment:
testnet_* ?

Reply:
I let test if it works

Reply:
Tested with this commit : https://github.com/massalabs/massa/pull/3367/commits/23f4078e2cd0a4f0107c1cd7906536037ac62023 it didn't saved anything in sanity job for example : https://github.com/massalabs/massa/actions/runs/3845249939/jobs/6549177614